stable/airflow/README.md

```diff
@@ -674,6 +675,7 @@ __Airflow Kubernetes Values:__
 | Parameter | Description | Default |
 | --- | --- | --- |
 | `rbac.create` | if Kubernetes RBAC resources are created | `true` |
+| `rbac.events` | if the airflow kubernetes role has get/list access to cluster events | `false` |
```

Comment:
Please use the same description I provided below:
```
if the created RBAC Role has GET/LIST on event Resources
```

Reply:
👍